jenkinsfiles/kubernetes-upstream.Jenkinsfile

```diff
@@ -36,7 +36,8 @@ pipeline {
         TESTDIR="${WORKSPACE}/${PROJ_PATH}/test"
         VM_MEMORY = "5120"
         K8S_VERSION="1.19"
-        SERVER_BOX = "cilium/ubuntu"
+        KERNEL="419"
```

Comment:
I'm commenting on the commit msg itself: Let's reference the commit that the comment that you are referring to, rather than posting a GH link. The reason is because GH links can be ephemeral, especially when referencing `master` rather than creating a "permalink".

Reply:
Makes sense, I have updated the commit message to include commit sha.